### PR TITLE
Create index.html

### PR DESCRIPTION
**Pull request by xritter1**
Link to Jira ticket if there is one: https://ctds-planx.atlassian.net/browse/PXP-9136

### Environments
qa-jcoin

### Description of changes
see ticket

### CODE REVIEW

**Comment by PlanXCyborg** [comment]
# qa-jcoin.planx-pla.net
 :x: etlMapping.yaml - gitops.json mismatch
- Field error: Field [project_id] in explorerConfig.filters.tabs.fields not found in etlMapping
- Field error: Field [project_id] in explorerConfig.table.fields not found in etlMapping
- Field error: Field [project_id] in explorerConfig.table.fields not found in etlMapping

**Comment by PlanXCyborg** [comment]
# qa-jcoin.planx-pla.net
 :x: etlMapping.yaml - gitops.json mismatch
- Field error: Field [project_id] in explorerConfig.filters.tabs.fields not found in etlMapping
- Field error: Field [project_id] in explorerConfig.table.fields not found in etlMapping
- Field error: Field [project_id] in explorerConfig.table.fields not found in etlMapping

**Comment by PlanXCyborg** [comment]
# qa-jcoin.planx-pla.net
 :x: etlMapping.yaml - gitops.json mismatch
- Field error: Field [project_id] in explorerConfig.filters.tabs.fields not found in etlMapping
- Field error: Field [project_id] in explorerConfig.table.fields not found in etlMapping
- Field error: Field [project_id] in explorerConfig.table.fields not found in etlMapping

File: qa-jcoin.planx-pla.net/portal/gitops.json [hunk 1]
```diff
@@ -724,5 +730,12 @@
     }
   ],
   "showArboristAuthzOnProfile": true,
-  "showFenceAuthzOnProfile": false
-}
\ No newline at end of file
+  "showFenceAuthzOnProfile": false,
+  "componentToResourceMapping": {
+    "Dashboard": {
+      "resource": "/secure_dashboard",
```
Comment (paulineribeyre) [inline qa-jcoin.planx-pla.net/portal/gitops.json:736]: the resource is `dashboard`
https://github.com/uc-cdis/commons-users/blob/506304c9130aec3b78916753e802321908ca16e0/users/qa-jcoin/user.yaml#L75

File: qa-jcoin.planx-pla.net/portal/gitops.json [hunk 2]
```diff
@@ -724,5 +730,12 @@
...
+      "method": "read",
+      "service": "dashboard"
```
Comment (paulineribeyre) [inline qa-jcoin.planx-pla.net/portal/gitops.json:738]: method is `access`
https://github.com/uc-cdis/commons-users/blob/506304c9130aec3b78916753e802321908ca16e0/users/qa-jcoin/user.yaml#L282-L283

**Comment by PlanXCyborg** [comment]
# qa-jcoin.planx-pla.net
 :x: etlMapping.yaml - gitops.json mismatch
- Field error: Field [project_id] in explorerConfig.filters.tabs.fields not found in etlMapping
- Field error: Field [project_id] in explorerConfig.table.fields not found in etlMapping
- Field error: Field [project_id] in explorerConfig.table.fields not found in etlMapping

**Review by paulineribeyre — APPROVED** [review]
could you update the PR title before merging

**Pull request merged**